### DISCUSSION TITLE
Show latest From/Correspondents in thread view when thread is collapsed

### DISCUSSION THREAD

**Discussion by soundfx37** [protz/thunderbird-conversations#1592] [Ideas]
* **Thunderbird version:** 78.6.0
* **Thunderbird Conversation version:** 3.2.10
* **Operating System:** Windows 10

**What did you do?** install Conversations, turn on threaded mode

**What did you expect to happen?** From/Correspondents columns show the first responder, not the last responder (most recent).  It is nearly impossible to keep track of which email is which with out knowing who I'm talking to.  I understand that you don't want to re organize the thread to last response on top, but if you just changed the From and or Correspondents columns to show the last responder that would be 10000x better and I think would serve as a nice compromise. 

**What actually happened?** From/Correspondents columns show original responder only.

**Comment by Standard8** [comment]
I've moved this across to an ideas discussion as I think that's more appropriate.

This is really referring to the message list view, which Conversations doesn't have any control over. Really this would be a Thunderbird thing to change. I did find one related bug: https://bugzilla.mozilla.org/show_bug.cgi?id=604033 - that doesn't really cover the from/correspondents, though the sentiment is the same.

We could potentially do this for the between column, though I've a feeling we would need Thunderbird support for it, as I don't think we get enough information for all the messages in the thread - only the top one.
